webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -112,22 +112,36 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
     this.subscriptions.add(subscription);
   }
 
+  private async recordSearchTelemetry(selectedContact, nextSelectedContact, nextFilters) {
+    if (!nextFilters?.search || !nextSelectedContact) {
+      return;
+    }
+
+    const hasSelectedNewContact = selectedContact === null || selectedContact._id !== nextSelectedContact._id;
+    if (!hasSelectedNewContact) {
+      return;
+    }
+
+    await this.searchTelemetryService.recordContactSearch(nextSelectedContact.doc, nextFilters.search);
+  }
+
   private subscribeToStore() {
-    const reduxSubscription = combineLatest(
+    const reduxSubscription = combineLatest([
       this.store.select(Selectors.getSelectedContact),
       this.store.select(Selectors.getForms),
       this.store.select(Selectors.getLoadingContent),
       this.store.select(Selectors.getLoadingSelectedContactReports),
       this.store.select(Selectors.getContactsLoadingSummary),
       this.store.select(Selectors.getFilters),
-    ).subscribe(([
+    ]).subscribe(([
       selectedContact,
       forms,
       loadingContent,
       loadingSelectedContactReports,
       contactsLoadingSummary,
       filters,
     ]) => {
+      this.recordSearchTelemetry(this.selectedContact, selectedContact, filters);
```

Comment:
We don't need to block the main thread by awaiting for this promise here, it can run in the background just fine

Reply:
I've only ever seen the `void` operator used in minified JS code to shorten `undefined` to `void 0` and gain 3 bytes for each occurrence :joy: 

But looking into your suggestion, it seems to be what typescript devs @ microsoft [recommend](https://github.com/microsoft/TypeScript/issues/13376#issuecomment-273289748) when you want to fire and forget a promise

+1 to start doing that